xml/dm_registers.xml

```diff
@@ -17,17 +17,16 @@
         time.  Eg. in a multi-hart system some might temporarily be powered
         down, or a system might support hot-swapping harts.
 
-        <field name="0" bits="31:23" access="R" reset="0" />
-        <field name="impebreak" bits="22" access="R" reset="Preset">
+        <field name="0" bits="31:24" access="R" reset="0" />
+        <field name="impebreak" bits="23" access="R" reset="Preset">
```

Comment:
Why are you moving impebreak?

Reply:
because dmerr moved it up if you really want 3 bits for it. I guess I missed the addition of dmerr, not sure what the point of it is, and also it seems weird to have it cross a byte boundary.